core/unit_test/TestAbort.hpp

```diff
@@ -103,5 +103,12 @@ void test_abort_from_device() {
 
 TEST(TEST_CATEGORY_DEATH, abort_from_device) {
   ::testing::FLAGS_gtest_death_test_style = "threadsafe";
+// FIXME_OPENACC FIXME_NVHPC: NVHPC fails when targetting CPUs.
+#if defined(KOKKOS_ENABLE_OPENACC) && defined(KOKKOS_COMPILER_NVHPC) && \
+    defined(KOKKOS_ENABLE_OPENACC_FORCE_HOST_AS_DEVICE)
+  GTEST_SKIP()
```

Comment:
I think we would want the
```C++
if (std::is_same_v<TEST_EXECSPACE, Kokkos::Experimental::OpenACC>)
```
because we still want to test the host backends regardless that the ACC backend is targeting the host

Reply:
Added.

Reply:
I meant just the occurrence you added not the other two that were already there